src/Microsoft.Health.SqlServer/Features/Schema/Manager/ISchemaManagerDataStore.cs

```diff
@@ -0,0 +1,25 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Microsoft.Health.SqlServer.Features.Schema.Manager
+{
+    public interface ISchemaManagerDataStore
```

Comment:
Could you add comments for the methods here and inheritdoc in SchemaManagerDataStore?

Reply:
Sure, done